sor/src/main/java/com/bazaarvoice/emodb/sor/core/DistributedCompactor.java

```diff
@@ -24,12 +25,12 @@ public DistributedCompactor(Counter archiveDeltaSizeInMemory, boolean keepDeltaH
         _legacyCompactor = new DefaultCompactor(archiveDeltaSizeInMemory, keepDeltaHistory, metricRegistry);
     }
 
-    public Expanded expand(Record record, long fullConsistencyTimestamp, long compactionConsistencyTimeStamp, MutableIntrinsics intrinsics,
+    public Expanded expand(Record record, long fullConsistencyTimestamp, long compactionConsistencyTimeStamp, long stashRunTimeStamp, MutableIntrinsics intrinsics,
```

Comment:
Consistency:  Here the timestamp is called "stashRunTimeStamp" but elsewhere it is also sometimes called "deleteDeltasTimestamp".  Recommend settling on one consistent name, maybe a middle ground between the two of something like "compactionControlTimestamp".